controllers/suite_test.go

```diff
@@ -48,7 +48,7 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = BeforeSuite(func(done Done) {
-	logf.SetLogger(zap.LoggerTo(GinkgoWriter, true))
+	logf.SetLogger(zap.New(zap.UseDevMode(true), zap.WriteTo(GinkgoWriter)))
```

Comment:
This can probably be reverted once you feel this PR is ready to go in

Reply:
Method `zap.LoggerTo` is deprecated. My PR includes updates, probably, worth to make a separate PR for it.

Reply:
Cool, but probably `UseDevMode`should be false?

Reply:
@eytan-avisror Made separate PR for that https://github.com/keikoproj/upgrade-manager/pull/44.
Also, old code uses `true`, I just kept same value.